source/easeOfAccess.py

```diff
@@ -10,12 +10,12 @@
 import winreg
 import ctypes
 import winUser
-from winVersion import winVersion
+import winVersion
 
 # Windows >= Vista
-isSupported = winVersion.major >= 6
+isSupported = winVersion.getWinVer().major >= 6

```

Comment:
Since we no longer support anything older than Win 7 this always evaluates to True. Have you considered removing this variable alltogether